app/assets/stylesheets/application.css.scss

```diff
@@ -36,13 +36,30 @@
   position: relative;
 }
 
+.trophy-case-container {
+  background-image: url('http://urlm.co/blog/wp-content/uploads/ipad23.jpg');
```

Comment:
Prefer double-quoted strings<br>URL `http://urlm.co/blog/wp-content/uploads/ipad23.jpg` should not contain protocol or domain